packages/cli-helpers/src/lib/__tests__/project.test.ts

```diff
@@ -58,6 +58,8 @@ describe('addEnvVar', () => {
         return true
       })
 
+      // @ts-expect-error We're only returning a string rather than the expected Buffer | string
+      // which is determined by the options parameter passed to the real fs.readFileSync
```

Comment:
@Tobbe I'm not happy to have added the `ts-expect-error` here but I spent far too long trying to get the types to be happy with this mock with no success.

Not sure if you have any ideas on getting TS to like this? If not we can always rework the structure of this test suite but I didn't want to do that here.

Reply:
@Josh-Walker-GM 
https://github.com/vitest-dev/vitest/issues/6182 
https://github.com/redwoodjs/redwood/pull/11032

Reply:
Awesome thanks! I'll keep an eye out for a patch release if they get one out.